src/WelcomeBody.css

```diff
@@ -0,0 +1,11 @@
+/** Scaffolding **/
+#welcomeBody {
+  grid-column: 1 / 3;
+  grid-template-rows: auto;
+  height: calc(100vh - 60px);
+}
+
+.panel-body {
+  overflow: auto;
+  text-align: center;
```

Comment:
Whats happening if .panel-body isn't applied?

Reply:
Since panel-body has text-aligment, if panel-body is removed, the text is left justified. I updated the CSS to have the text alignment in welcome-screen element and removed panel-body.